doc/nrf/ug_nrf91_features.rst

```diff
@@ -79,7 +79,7 @@ See also :ref:`tfm_hello_world` for a sample that demonstrates how to add TF-M t
 Application
 -----------
 
-The user application runs in the non-secure domain.
+The user application runs in NSPE.
```

Comment:
`user application` 
`main application`
`main user application` 

I think all these three terms are used for the same concept, we should consider aligning.

Reply:
Aligned to "user application".